scripts/generator/graphql_generator/csharp/SDK/Serializable.cs.erb

```diff
@@ -1,14 +1,9 @@
-#if !SHOPIFY_MONO_UNIT_TEST
 namespace <%= namespace %>.SDK {
     using System;
     using System.Collections;
     using System.Collections.Generic;
-    using UnityEngine;
 
-    public class Serializable {
-        public string ToJsonString() {
-            return JsonUtility.ToJson(this);
-        }
+    public interface Serializable {
+        string ToJsonString();
```

Comment:
I wonder if we should add in something like:
`object ToJson();`
`ToJsonString` could use the result of `ToJson`